test/fuzz/mempool/checktx.go

```diff
@@ -0,0 +1,34 @@
+package checktx
+
+import (
+	"github.com/tendermint/tendermint/abci/example/kvstore"
+	"github.com/tendermint/tendermint/config"
+	mempl "github.com/tendermint/tendermint/mempool"
+	"github.com/tendermint/tendermint/proxy"
+)
+
+var mempool mempl.Mempool
+
+func init() {
+	app := kvstore.NewApplication()
+	cc := proxy.NewLocalClientCreator(app)
+	appConnMem, _ := cc.NewABCIClient()
+	err := appConnMem.Start()
+	if err != nil {
+		panic(err)
+	}
+
+	cfg := config.DefaultMempoolConfig()
+	cfg.Broadcast = false
+
+	mempool = mempl.NewCListMempool(cfg, appConnMem, 0)
+}
+
+func Fuzz(data []byte) int {
```

Comment:
Why use a global and `init()` instead of just inline here?

Reply:
If I am not mistaken, `Fuzz` function is executed in a cycle by https://github.com/dvyukov/go-fuzz. I could put everything in `Fuzz`, but then FOR EACH random bytes sequence (generated by fuzzier) we will be creating mempool, which I think is slower. TLDR: it's for speed